rest-tests/src/test/groovy/org/hawkular/metrics/rest/CassandraBackendITest.groovy

```diff
@@ -34,6 +34,66 @@ class CassandraBackendITest extends RESTTest {
   }
 
   @Test
+  void findMetricsWhenThereIsNoData() {
```

Comment:
I spent a whole lot of time my import settings last Friday. The settings for static imports were not configured in the build tools repo. I think getting the settings aligned is still a bit a work in progress.